### ISSUE TITLE
Welcome: Loss of focus after "More" button in the editor title

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#53145]
**Environment Details:** 
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
MAS Violated: MAS2.4.3

**Repro Steps:**
1. Launch VS Code
2. Click on Help from the menu bar--> Click on Welcome
3. Press tab key till More button.

**Actual:**
Loss of focus found after "More" button and before New File link.

**Expected:**
After More Button, focus should move to new File link.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**MAS Reference**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

Attachment for Reference: 
![issu5](https://user-images.githubusercontent.com/39204909/41974793-8f12aa3c-7a36-11e8-93c3-fc2e34753789.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Invalid. After more we focus the whole window, and after that we focus the first item in the weclome again. Just like in any other editor.
This is just how focus works in the browsers and this is expected behavior.
fyi @bpasero
